bazel/conf/.bazelrc.build

```diff
@@ -75,10 +75,12 @@ test:testnet --test_output=streamed --test_tag_filters=
 # TODO(IDX-2374): enable alltests in CI when we will have actual system tests.
 #test:ci --config=alltests
 
-# Run all tests once by default, run flaky tests up to 3 times in CI.
+# Run all tests once by default.
 test    --flaky_test_attempts=1
```

Comment:
I'm guessing the rationale here is that locally you probably want a flaky test to fail explicitly on the first failure and not be retried? Would it make sense to reflect this in the comment above?